scout/server/blueprints/variant/views.py

```diff
@@ -21,14 +21,24 @@
 )
 
 from scout.parse.clinvar import set_submission_objects
-
 from scout.constants import ACMG_CRITERIA, ACMG_MAP
 
 LOG = logging.getLogger(__name__)
 
 variant_bp = Blueprint("variant", __name__, static_folder="static", template_folder="templates")
 
 
+@variant_bp.route("/update_tracks", methods=["POST"])
+def update_tracks_settings():
+    """Update custom track settings for a user according to form choices"""
+    user_obj = store.user(email=current_user.email)
+    selected_tracks = request.form.getlist("user_tracks") or []
+    # update user in database with custom tracks info
+    user_obj["igv_tracks"] = selected_tracks
```

Comment:
Perhaps add to User object model as well, and init with something sensible for new users?

Reply:
But if I add all available tracks to the user object when a user is created, then if we add one or more selectable tracks later, these will not be selected for those users, and perhaps they won't know that they even exist if they don't go into the settings and change them manually. Isn't it better if the key doesn't exist of is None so that all available tracks are always loaded if a user doesn't save any track preference?

Reply:
Wouldn’t the same be true as soon as they save settings once, with the defaults being implicit in this function rather than on user init? How about making new tracks show, eg only hiding ones that have explicitly been marked hidden - not ones that didn’t exist before?

Reply:
Absolutely, but looks quite complicates to handle

Reply:
Generally I'd say we want to have all stored elements on objects in the class / object model. There may be exceptions. Not documenting them there seems like a recipe for confusion, e.g. when serialising objects?

In this case that would ensure new users get sensible defaults, but old users must choose once. Doesn't that make sense?

Reply:
Sure, I'll add all general tracks to the newly created users.

Reply:
Done, and I've updated the "build_user" test to check that the default tracks are created when a new user is created.